### ISSUE TITLE
The IWC website is either broken or incorrect in the README

### ISSUE THREAD

**Issue by danhooper** [aml-development/ozp-iwc#398]
I could not load https://ozone-development.github.io/ozp-iwc/ which is linked from the documentation.

**Comment by Kevin-K** [comment]
Correct, I'm no longer directly affiliated with the development of the `ozp-iwc` but it seems two things have occurred.

1) There was a rebranding of the organization from `ozp-development` to `aml-development`.

2) new releases were pushed but there were steps missed on updating the gh-page website.

I'll take a look through and see what needs to be changed.

**Comment by Kevin-K** [comment]
The release differences between `1.2.2` and `1.2.4` are actually all identical, they are just version bumps, until the gh-pages is updated to `1.2.4` assume `1.2.2` docs are **latest**.

The tutorial/examples are broken currently because the code samples are all hosted on [codepen.io](codepen.io) and reference the old `ozp-development`. I'm updating those as I write this, I'll note on this thread when they are fixed.

There are a hand full of example links that are hosted in the [aml-development/ozp-demo](https://github.com/aml-development/ozp-demo) gh-pages as well which need redirecting.

**Comment by chazzlabs** [comment]
@Kevin-K Thanks for being so responsive regarding the issues here lately, especially since they're not necessarily referencing bugs but rather documentation inconsistencies.

**Comment by rpokorny** [comment]
Please see this google group announcement: https://groups.google.com/d/msg/ozoneplatform-users/y_skbS58Y7w/d99n15dLCAAJ

Per that announcement, the "canonical" copies of the OZP repos, including ozp-iwc, now live in the https://github.com/ozoneplatform group.  This means that the canonical copy of the IWC documentation now lives at https://ozoneplatform.github.io/ozp-iwc .  I believe that I had fixed all links on that copy, but I did not adjust the version to `1.2.4`.  @Kevin-K , if you could show me how to do that that would be great.

**Comment by chazzlabs** [comment]
@rpokorny Thanks.  Was this announcement referenced anywhere else?  Our team's been using the AML Development repos and didn't know about this change.

**Comment by Kevin-K** [comment]
@rpokorny there are some instructions on the [gh-pages README](https://github.com/aml-development/ozp-iwc/blob/gh-pages/README.md). Ideally that should all be some build automated operation, but at the time of making the site I was fairly short-handed. 

I'm in the process of updating those links on my [fork](https://github.com/kevin-k/ozp-iwc/tree/gh-pages) and will put up a PR with the changes.

***

Continuing discussion on the ozoneplatform vs aml-development here for search-ability:
>Per that announcement, the "canonical" copies of the OZP repos, including ozp-iwc, now live in the https://github.com/ozoneplatform group. 

Reading through the announcement, it sounds that `ozoneplatform` github org retains the FOSS releases & maintenance, while the `aml-development` github org is a "downstream" effort that can contribute back to `ozoneplatform` where seen fit. 

If thats correct, wouldn't it make more sense that the `ozoneplatform` organization hold the repositories and `aml-development` holds forks? 

It seems the current structure draws questions/issues to the aml-development repo's inadvertently.
